chalicelib/responseobjects/hca_response_v5.py

```diff
@@ -122,26 +125,52 @@ class ManifestResponse(AbstractResponse):
     """
     Class for the Manifest response. Based on the AbstractionResponse class
     """
+
     def return_response(self):
-        return self.apiResponse
+        es_search = self.es_search
+
+        def field_mapper():
+            for hit in es_search.scan():
+                hit_dict = hit.to_dict()
+                mapped_dict = {}
+                for friendly_name_key, es_name_value in self.mapping.iteritems():
+                    es_val = jmespath.search(es_name_value, hit_dict)
```

Comment:
This compiles the JMESpath for every result item which is what we wanted to avoid. Google "jmespath compile" for details.